### PR TITLE
Design recipients screen

### PR DESCRIPTION
**Pull request by wrightmartin**
This makes ```manage/recipients/``` much easier to digest

### CODE REVIEW

**Comment by wrightmartin** [comment]
screen shot 2015-03-30 at 16 28 03

File: nuntium/templates/nuntium/profiles/contacts/contacts-per-writeitinstance.html [hunk 1]
```diff
@@ -38,24 +38,24 @@
   <div class="page-header">
     <h2>{% trans "Recipients" %}</h2>
   </div>
-
+  <p>{% trans "There are" %} <strong>{{ writeitinstance.persons.count }}</strong> {% trans "recipients in your WriteIt" %}</p>
```
Comment (tmtmtmtm) [inline nuntium/templates/nuntium/profiles/contacts/contacts-per-writeitinstance.html]: I'm assuming there's a better way to do this that will work in languages that form a sentence like this differently? (and to cope with the case of only having one person)

Comment (wrightmartin) [inline nuntium/templates/nuntium/profiles/contacts/contacts-per-writeitinstance.html]: @tmtmtmtm almost definitely, if a dev has two seconds I'm sure they could do this 1000% better

Comment (chrismytton) [inline nuntium/templates/nuntium/profiles/contacts/contacts-per-writeitinstance.html]: Yes, you can use [`{% blocktrans %}`](https://docs.djangoproject.com/en/1.6/topics/i18n/translation/#std:templatetag-blocktrans) to handle variables in language sentences and pluralization:

```
{% blocktrans with count person_count=writeitinstance.persons.count %}
There is 1 recipient in your WriteIt
{% plural %}
There are {{ person_count }} recipients in your WriteIt
{% endblocktrans %}
```

Comment (wrightmartin) [inline nuntium/templates/nuntium/profiles/contacts/contacts-per-writeitinstance.html]: amazing, thanks chris

**Comment by chrismytton** [comment]
:+1:

**Pull request merged**